docs/apache-airflow-providers-microsoft-azure/operators/s3_to_azure_blob.rst

```diff
@@ -0,0 +1,61 @@
+
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+AWS S3 to Azure Blob Storage Transfer Operator

```

Comment:
Shouldn't the doc be in `docs/apache-airflow-providers-microsoft-azure/transfers/s3_to_azure_blob.rst` instead of `docs/apache-airflow-providers-microsoft-azure/operators/s3_to_azure_blob.rst`

Reply:
After merging #32241, i think we can move above to the transfers folder.

Reply:
Agreed. @RachitSharma2001 would you mind updating the doc path?